Gordon360/Services/HousingService.cs

```diff
@@ -93,28 +93,26 @@ public bool RemoveHousingAdmin(string id)
         /// Calls a stored procedure that tries to get the id of an the application that a given user is 
         /// applicant on for a given session
         /// </summary>
-        /// <param name="studentId"> The id to look for </param>
+        /// <param name="studentID"> The id to look for </param>
         /// /// <param name="sess_cde"> Session for which the application would be </param>
         /// <returns> 
         /// The id of the application or 
         /// null if the user is not on an application for that session 
         /// </returns>
-        public int? GetApplicationID(string studentId, string sess_cde)
+        public int? GetApplicationID(string studentID, string sess_cde)
         {
-            IEnumerable<ApartmentAppIDViewModel> idResult = null;
+            IEnumerable<AprtAppID> idResult = null;
```

Comment:
I thought this got switched back... Evan Platzer commented about ApartmentAppIDViewModel in the load PR, this is an ongoing issue I am trying to solve. 
We should not need a full view model for this SQL query, and if we did we would just use one autogenerated with the EDMX. 
So it's not yet solved

Reply:
Oh, I see... :(

Reply:
Oh, wait could we not use a view model that has the application ID variable among other vars?

Reply:
We could, but the column naming convention of the DB would need to match the C# naming convention, and still it is sort-of a risky and error-prone way of trying to get that to work. 

There seem to be ways of returning just the singular int directly to a variable, I haven't researched it far enough yet to actually implement it. It gets weird because it looks like all the SQL query commands will also use an int to return a success or fail or error codes, so you have to make sure you are actually returning the value from the result of the query, and not an int representing the error code by mistake....

Needs more research